notebooks/Datasources.ipynb

```diff
@@ -0,0 +1,393 @@
+    "# Neomaril DataSources\n",
+    "\n",
+    "This notebook give a exemple on how to import your datasorces and datasets to Neomaril"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### NeomarilDataSourceClient\n",
+    "\n",
+    "It's where you can manage your Datasources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the client\n",
+    "from neomaril_codex.datasources import NeomarilDataSourceClient"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:19:35.385 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:19:37.219 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<neomaril_codex.datasources.NeomarilDataSourceClient at 0x7f6ba047ded0>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Start the client. We are reading the credentials in the NEOMARIL_TOKEN env variable\n",
+    "client = NeomarilDataSourceClient()\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### NeomarilDataSource\n",
+    "\n",
+    "It's where you can register your datasource."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:19:43.230 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:19:43.238 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n",
+      "2024-03-20 19:19:44.777 | INFO     | neomaril_codex.datasources:register_datasource:101 - DataSource 'testeDataSouce' was registered!\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<neomaril_codex.datasources.NeomarilDataSource at 0x7f6ba047e170>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "credentials_path = os.path.abspath('./samples/datasources/credentials.json')\n",
+    "\n",
+    "client.register_datasource(\n",
+    "    datasource_name='testeDataSouce',\n",
+    "    provider='GCP',\n",
+    "    cloud_credentials=credentials_path,\n",
+    "    group='datarisk'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List Datasources\n",
+    "\n",
+    "Use this function to find your datasouce."
```

Comment:
Consider to change for something like "Use this function to find all data sources in your group from one specific provider"